app/views/volunteers/_form.html.slim

```diff
@@ -114,6 +118,8 @@
     .col-xs-12
       = f.input :region, collection: Volunteer.region_collection, as: :radio_buttons, label: false
 
+  = render 'schedules/fields', f: f, participant: @volunteer
```

Comment:
Change the naming of this partial to the same as the others: `= render 'schedules/form', f: f, participant: @volunteer`